introduction.md

```diff
@@ -0,0 +1 @@
+git add introduction.md "Fuck Fuck"
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.